src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -207,4 +208,28 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('date')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('format')->defaultNull()->end()
+                        ->scalarNode('interval_format')->defaultNull()->end()
+                        ->scalarNode('timezone')->defaultNull()->end()
+                    ->end()
+                ->end()
+                ->arrayNode('number_format')
```

Comment:
should add info that this section configures the default behavior of the `number_format` filter